apps/prairielearn/elements/pl-number-input/number_input_test.py

```diff
@@ -0,0 +1,77 @@
+import importlib
+
+import pytest
+
+number_input = importlib.import_module(
+    "pl-number-input"
+)  # TODO To Reviewer: is there better way to import?
```

Comment:
Can we drop this TODO before this PR is merged?

Reply:
Removed here: https://github.com/PrairieLearn/PrairieLearn/pull/8435/commits/682198bbf6fc86d79dacbfee1c8418b6ba91cd64